pelicun/model/pelicun_model.py

```diff
@@ -313,28 +309,31 @@ def _get_locations(self, loc_str: str) -> np.ndarray:
         array(['11'])
         """
         try:
-            res = str(int(loc_str))
+            res = str(int(float(loc_str)))
             return np.array([res])
 
         except ValueError as exc:
             stories = self._asmnt.stories
 
             if "--" in loc_str:
                 s_low, s_high = loc_str.split('--')
-                s_low = self._get_locations(s_low)
-                s_high = self._get_locations(s_high)
+                s_low = self._get_locations(s_low)[0]
+                s_high = self._get_locations(s_high)[0]
```

Comment:
You are right. This could be very serious, as the indexing in the following line would result in getting the first charachter of a string representing location, which would be pretty bad. I removed the redundant indexing operations with e60e0c47.

Reply:
Two things:
- You seem to have left a breakpoint in the committed code -> I see you fixed this in a later commit.
- I wonder why this error did not break any of our tests. I assume we don't have a `--` in any of the location assignments in our tests. Could you add it to one just to be more protected from similar bugs in the future?